src/test/java/core/basesyntax/service/impl/MyFileWriterCsvImplTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.MyFileReader;
+import core.basesyntax.service.MyFileWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class MyFileWriterCsvImplTest {
+    private static final String TEST_FRUIT_WHICH_LOCK_CSV
+            = "src/test/resources/test-file-which-lock.csv";
+    private static final String TEST_FRUIT_REPORT_CSV
+            = "src/test/resources/test-report-fruit-shop.csv";
+    private static MyFileWriter myFileWriterCsv;
+    private static MyFileReader myFileReaderCsv;
+    private static String dataForWrite;
+    private static final String emptyDataForWrite = "";
+
+    @BeforeClass
+    public static void beforeClass() {
+        myFileWriterCsv = new MyFileWriterCsvImpl();
+        myFileReaderCsv = new MyFileReaderCsvImpl();
+        dataForWrite = "type,fruit,quantity" + System.lineSeparator()
+                + "b,banana,20" + System.lineSeparator()
+                + "b,apple,100" + System.lineSeparator();
+    }
+
+    @Test
+    public void writeToFileUsualData_Ok() {
+        myFileWriterCsv.writeToFile(TEST_FRUIT_REPORT_CSV, dataForWrite);
+        List<String> expected = Arrays.asList("type,fruit,quantity","b,banana,20", "b,apple,100");
+        List<String> actual = myFileReaderCsv.readFromFile(TEST_FRUIT_REPORT_CSV);
```

Comment:
let's not use our custom reader service in the test where we are testing writer
our result starts depending on the reader as well

you can just use Files.readAllLines() or the equivalent